### PR TITLE
Add ability use external baremetal-operator configs

### PR DESCRIPTION
**Pull request by jingvar**
Introduce external config files and entry points for containers in baremetal-operator.  It can be applied to containers witch require custom configuration parameters.

### CODE REVIEW

File: charts/baremetal-operator/templates/ironic.yaml [hunk 1]
```diff
@@ -100,14 +106,21 @@ spec:
           volumeMounts:
               - mountPath: "/shared"
                 name: ironic-storage
-              {{- if .Values.ironic.config_override.httpd }}
+              {{- if or .Values.ironic.config_override.httpd .Values.ironic.config_override.inspector_ipxe }}
               - name: httpd-entrypoint
                 mountPath: /bin/runhttpd
                 subPath: runhttpd.sh
+              {{- end }}
+              {{- if or .Values.ironic.config_override.httpd }}
```
Comment (dukov) [inline charts/baremetal-operator/templates/ironic.yaml]: remove 'or'

**Review by dukov — CHANGES_REQUESTED** [review]

File: charts/baremetal-operator/templates/ironic.yaml (hunk 1 above)
Comment (jingvar) [reply charts/baremetal-operator/templates/ironic.yaml]: Done

**Review by dukov — APPROVED** [review]

**Pull request closed without merging**